go.mod

```diff
@@ -87,12 +87,12 @@ require (
 	github.com/pkg/profile v1.2.1 // indirect
 	github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a // indirect
 	github.com/rs/cors v1.5.0 // indirect
-	github.com/sirupsen/logrus v1.3.0 // indirect
```

Comment:
I don't know why this happened.

Reply:
i also don't. it's okay though

Reply:
maybe just `git add -p` to not add this removal to your commit but add the zipfs one only?